### DISCUSSION TITLE
Example of configuration with Angular Workspace?

### DISCUSSION THREAD

**Discussion by EvanCarroll** [semantic-release/semantic-release#2022] [Configuration] [answered]
One Angular workspace can contain many "projects". How can I configure it such that my workspace can publish more than one packages? How do I differentiate commits by workspace (is there a method to specify project). Does the project support commits that cross multiple projects?

Example,

```sh
ng new --create-application=false --skip-install foobar
cd foobar
ng g lib quz --skip-install
ng g lib qaz --skip-install
```

**Comment by travi** [comment, marked as answer]
semantic-release does not officially support monorepos. there are a few tools that add very specific support for such structures, like `semantic-release-monorepo`, but those tools are outside of the scope of official support
